src/App.css

```diff
@@ -0,0 +1,39 @@
+.App {
```

Comment:
- [ ] As the requirement given we need to delete all. I can see that this app css is the base styling that need to be removed. Kindly remove it :)